README.md

```diff
@@ -55,24 +79,65 @@ You can also set following mount points by passing `-v /host:/container` flag to
 | :----------------------- | ------------------------------------- |
 |  `/var/lib/pgsql/data`   | PostgreSQL database cluster directory |
 
-## Usage
 
-We will assume that you are using the `openshift/postgresql-92-centos7` image. Supposing that you want to set only mandatory environment variables and not store the database in a host directory, you need to execute the following command:
+Usage
+----------------------
 
-```console
-docker run -d --name postgresql_database -e POSTGRESQL_USER=user -e POSTGRESQL_PASSWORD=pass -e POSTGRESQL_DATABASE=db -p 5432:5432 openshift/postgresql-92-centos7
+We will assume that you are using the `openshift/postgresql-92-centos7` image.
+Suppose that you want to set only mandatory environment variables and not store
+the database in a host directory, you need to execute the following command:
```

Comment:
I'm actually not sure! :frowning: 

From a little googling it seems they are actually pretty interchangeable:
http://forum.wordreference.com/showthread.php?t=520484

You could replace suppose/supposing with "If" and simplify it :)  ("If you want to......")